.travis.yml

```diff
@@ -50,6 +53,8 @@ before_install:
   - travis_retry sudo apt-get update -qq
   - git config --global user.email "test@travis.land"
   - git config --global user.name "Travis Almighty"
+  # Setup SSH docker service
+  - docker run -d -p 49000:22 rastasheep/ubuntu-sshd:14.04
```

Comment:
I think travis does not support docker ... we would need to use circle-ci if we decide to test against live docker instances